docs/source/daml/intro/7_Composing.rst

```diff
@@ -0,0 +1,271 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+7 Composing Choices
+===================
+
+It is time to put everything you have learnt so far together into a complete and secure DAML model for asset issuance, management, transfer, and trading, with capabilities similar to that presented in :doc:`/getting-started/quickstart`. In the process you will learn about a few more concepts:
+
+- DAML Projects, Packages and Modules
+- Composition of transactions
+- Observers and Stakeholders
+- DAML's execution model
+- Privacy
+
+The model in this section is not a single DAML file, but a DAML project consisting of several files that depend on each other.
+
+DAML Projects
+-------------
+
+DAML is organized in packages and modules. DAML projects are specified using a ``daml.yaml`` file and compile into packages. DAML files within a project become DAML modules. You can start a new project with a skeleton structure using ``daml new project_name`` in the terminal.
```

Comment:
I've clarified this but let me know if it's not actually correct, as the sense did change a little

```suggestion
DAML is organized in packages and modules. A DAML project is specified using a single ``daml.yaml`` file, and compiles into a package. Each DAML file within a project becomes a DAML module. You can start a new project with a skeleton structure using ``daml new project_name`` in the terminal.
```